kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/Util.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.kubernetes.operator;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
+
+/**
+ * Constants and methods used by multiple other classes.
+ */
+public class Util {
```

Comment:
I'm going to voice my usual grumble about Util classes being a smell about object denial and lack of a proper home for logic and move on. 

I can't summon up the energy to write a Junit extension to supply a properly configured object mapper.

Reply:
Not sure I follow your object denial point is TBH. Its just a function. What would you have me do instead?

Reply:
Probably just new up the mapper in the tests but this is fine.

Utils classes are often a way of sharing logic between various call sites and thus tend to mark shared behaviour (so far so uncontentious) but they tend to bundle up behaviour and flags over time. I guess my object denial point is from a DDD perspective a lot of that behaviour should be wrapped in an object that understands the business rules.

https://mattilehtinen.com/articles/dunghill-anti-pattern-why-utility-classes-and-modules-smell/ seems a reasonable summary of my views though I'm sure I read something more persuasive at some point.